feature/gribi/otg_tests/ipv4_entry_test/ipv4_entry_test.go

```diff
@@ -297,8 +297,13 @@ func TestIPv4Entry(t *testing.T) {
 		},
 		{
 			// ate port link cannot be set to down in kne, therefore the downPort is a dut port
-			desc:     "Downed next-hop interface",
-			downPort: dut.Port(t, "port2"),
+			desc: "Downed next-hop interface",
+			downPort: func() *ondatra.Port {
+				if deviations.ATEPortLinkStateOperationsUnsupported(ate) {
```

Comment:
These will apply to ate.Devices in KNE, but those do not actually include model/version information currently. I will be making changes to adding defaults for KNE ATE models, but since I don't know the final format I'm not adding the metadata changes in this patch. (The updates in this test should be the default behavior anyways.)